DOCS-2382: Update readme (#222)
viamrobotics/viam-flutter-sdk: README.md

`flutter pub add viam_sdk`

## Upgrading

`flutter pub upgrade viam_sdk`

> Note that breaking changes may occur in patch version changes, so we recommend locking your project to a specific version by adding a specific version number (for example `viam_sdk: 0.0.12`) to your `pubspec.yaml` file.

Reviewer: I would mention that breaking changes may occur in patch version changes BECAUSE we're still in beta.

Author: ```suggestion
Note that the TypeScript SDK is still in beta, so breaking changes may occur in patch version changes.
We recommend locking your project to a specific version by adding a specific version number (for example `viam_sdk: 0.0.12`) to your `pubspec.yaml` file.
```

`flutter pub add viam_sdk`

## Upgrading

`flutter pub upgrade viam_sdk`

> Note that breaking changes may occur in patch version changes because the TypeScript SDK is still in beta.

Reviewer 2: ```suggestion
Note that breaking changes may occur in patch version changes because the Viam Flutter SDK is still in beta.
```

> Note that breaking changes may occur in patch version changes because the TypeScript SDK is still in beta.

Author: ```suggestion
Note that breaking changes may occur in patch version changes because the Flutter SDK is still in beta.
```
